modules/core/04-channel/keeper/handshake_test.go

```diff
@@ -258,22 +199,14 @@ func (suite *KeeperTestSuite) TestChanOpenTry() {
 			channelKey := host.ChannelKey(counterparty.PortId, counterparty.ChannelId)
 			proof, proofHeight := suite.chainA.QueryProof(channelKey)
 
-			channelID, capability, err := suite.chainB.App.GetIBCKeeper().ChannelKeeper.ChanOpenTry(
+			_, err := suite.chainB.App.GetIBCKeeper().ChannelKeeper.ChanOpenTry(
```

Comment:
probably could assert that the channelID is non-empty in the success case, but fine as is as well